src/runtime_src/driver/zynq/drm/zocl/zocl_drv.c

```diff
@@ -134,6 +134,26 @@ static int get_reserved_mem_region(struct device *dev, struct resource *res)
 	return 0;
 }
 
+/**
+ * is_valid_apts - Check the geiven phys address is the start of a aperture
+ *
+ * @dev: DRM device struct
+ * @size: This size was not use, just to match function prototype.
+ *
+ */
+int is_valid_apts(struct drm_zocl_dev *zdev, phys_addr_t addr)
```

Comment:
Nit: I suggest using a function name something like get_apt_num(). Normally is_valid() returns true or false and can be used directly in if statement.

Reply:
Good suggestion, will do.